src/test/java/ru/training/at/hw3/main/homepage/pagecomponents/HeaderSection.java

```diff
@@ -0,0 +1,40 @@
+package ru.training.at.hw3.main.homepage.pagecomponents;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import ru.training.at.hw3.test.TestsInitialization;
+
+public class HeaderSection {
+
+    @FindBy(xpath = "//a[@href='index.html']")
+    public WebElement homeButton;
+
+    @FindBy(xpath = "//a[@href='contacts.html']")
+    public WebElement contactFormButton;
+
+    @FindBy(xpath = "//a[@class='dropdown-toggle']")
+    public WebElement serviceButton;
+
+    @FindBy(xpath = "//a[@href='metals-colors.html']")
+    public WebElement metalsAndColorsButton;
+
+    @FindBy(xpath = "//a[text()='Different elements']")
+    public WebElement differentElementsButton;
+
+    public HeaderSection() {
+        PageFactory.initElements(TestsInitialization.driver, this);
+    }
+
+    public void checkHeaderMenu() {
+        Assert
```

Comment:
Similar story about collections. For example you have Expected constant - List: HOME, CONTACT FORM, SERVICE, METALS in separate class. You could define List<WebElement> with selector 'ul.uui-navigation.navbar-nav > li' and in method in page object return outside List<String> - texts of those menus: 

    @FindBy(css = "ul.uui-navigation.navbar-nav > li")
    List<WebElement> headerElements;

    public List<String> getHeaderElementsTexts() {
        return headerElements.stream().map(WebElement::getText).collect(Collectors.toList());
    }

Reply:
But if you want to click on particular element you would find it in collection using streams as well. So all selectors in that class (and in similar) could replaced with just one List of elements, if it's comfortable you could use specific selector for elemnt in dropdown menu ('Different Elements Page'), it's because of homework (but in general it will be more useful to describe all possibilities of that menus in a few methods, it could be simple and flexible).